### PR TITLE
109-spatial-flip

### LINKED ISSUE #109: Port spatial flip/mirroring transform
a subtask of #62, port a spatial flipping/mirroring transform into Monai

ref:
https://docs.scipy.org/doc/numpy/reference/generated/numpy.flip.html

### PR DESCRIPTION
**Pull request by madil90**
Fixes #109  .

### Description
Adds Flip transform. Tests coming soon.

### Status
**Work in progress**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### CODE REVIEW

**Comment by madil90** [comment]
@wyli added tests.

File: monai/transforms/transforms.py [hunk 1]
```diff
@@ -72,12 +72,12 @@ class Flip:
     """
 
     def __init__(self, axes=None):
-        assert isinstance(axes, int) or isinstance(axes, tuple) and all(isinstance(n, int) for n in axes), \
+        assert isinstance(axes, (int, list, tuple)) and all(isinstance(n, int) for n in axes), \
```
Comment (Nic-Ma) [inline monai/transforms/transforms.py]: I think this line is still not correct:
```py
all(isinstance(n, int) for n in axes)
```
If axes is an int number, it's **not iterable**, your unit test doesn't cover int number case.

File: monai/transforms/transforms.py [hunk 2]
```diff
@@ -72,12 +72,12 @@ class Flip:
     """
 
     def __init__(self, axes=None):
```
Comment (Nic-Ma) [inline monai/transforms/transforms.py]: Why you allow a default value of **axes**=None?
I think we must ask the user to set axes, do you agree?
Thanks.

Comment (madil90) [reply monai/transforms/transforms.py]: By default, it flips over all the axes.

File: monai/transforms/transforms.py (hunk 1 above)
Comment (madil90) [reply monai/transforms/transforms.py]: Fixed.

**Pull request merged**